src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/form/button/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:Component"
     jcr:title="Form Button"
     sling:resourceSuperType="core/wcm/components/form/button/v2/button"
-    componentGroup="${componentGroupName} Form"/>
+    componentGroup="${componentGroupName}.Content Form"/>
```

Comment:
shouldn't these be `${componentGraoupName}.form`?

Reply:
The goal for naming is to align it with the WKND tutorial (see for example https://github.com/adobe/aem-guides-wknd/blob/develop/ui.content/src/main/content/jcr_root/conf/wknd/settings/wcm/policies/.content.xml#L106)

Reply:
see naming comment above

Reply:
ok, makes sense to align, but the grouping in WKND is strange to me, given the project structure:
```
/components/form
/components/content
/components/structure
```

The `myComponentGroup.Content Form` naming is a little confusing to me, if it's really a subgroup shouldn't it be `myComponentGroup.Content.Form`.